### PR TITLE
[docs] task: Add guide on how to use Ledger with Symbol Desktop Wallet

### PR DESCRIPTION
**Pull request by cryptoBeliever**
### What's the issue?

Currently, there is no official Ledger device set-up guide in docs.

### What's the fix?

Add a guide on how to use the Ledger device with Symbol Desktop Wallet.

### CODE REVIEW

File: source/guides/account/how-to-use-ledger-hardware-wallet.rst [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+  :align: center
+  :width: 100%
+  :class: with-shadow
+  :target: /_images/ledger-guide-12.png
+
+4. Review each transaction detail in your Ledger device and press the right button to continue.
+
+* Recipient’s Symbol address
+
+.. figure:: ../../resources/images/screenshots/ledger-guide-13.png
```
Comment (Wayonb) [inline source/guides/account/how-to-use-ledger-hardware-wallet.rst:199]: It looks like this should be image 14 for the address and image 13 is the transaction type?

**Review by Wayonb — CHANGES_REQUESTED** [review]

File: source/guides/account/how-to-use-ledger-hardware-wallet.rst [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+
+***************
+Contact Support
+***************
+
+For technical support, you may reach us at:
+
+- Discord: https://discord.com/invite/xymcity
+
+- Telegram: @nemhelpdesk
```
[the remark this answers is not in the document]
Comment (cryptoBeliever) [reply source/guides/account/how-to-use-ledger-hardware-wallet.rst:305]: Telegram @nemhelpdesk is for NEM and Symbol.

File: source/guides/account/how-to-use-ledger-hardware-wallet.rst [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+
+5. Ten accounts will be presented. Please choose at least one of them to continue. All accounts are derived from your Ledger device.
+
+.. figure:: ../../resources/images/screenshots/ledger-guide-5.png
+  :align: center
+  :width: 100%
+  :class: with-shadow
+  :target: /_images/ledger-guide-5.png
+
+5. To confirm profile creation accept the Terms and click the **Next** to continue. Note that all transactions must be confirmed via your Ledger device NOT with your password.
```
Comment (Wayonb) [inline source/guides/account/how-to-use-ledger-hardware-wallet.rst:107]: There is two step 5.

File: source/guides/account/how-to-use-ledger-hardware-wallet.rst [hunk 4]
```diff
@@ -0,0 +1,305 @@
...
+
+* Amount
+
+.. figure:: ../../resources/images/screenshots/ledger-guide-16.png
+  :align: center
+  :width: 100%
+  :class: with-shadow
+  :target: /_images/ledger-guide-16.png
+
+* Message
```
Comment (Wayonb) [inline source/guides/account/how-to-use-ledger-hardware-wallet.rst:219]: should there be message type and message text?

**Review by Wayonb — CHANGES_REQUESTED** [review]

File: source/guides/account/how-to-use-ledger-hardware-wallet.rst (hunk 3 above)
Comment (cryptoBeliever) [reply source/guides/account/how-to-use-ledger-hardware-wallet.rst]: Thanks, changed.

File: source/guides/account/how-to-use-ledger-hardware-wallet.rst (hunk 4 above)
Comment (cryptoBeliever) [reply source/guides/account/how-to-use-ledger-hardware-wallet.rst]: ok, splitted into two points.

**Review by Wayonb — APPROVED** [review]

**Pull request merged**